src/app/zap-templates/zcl/data-model/chip/operational-state-rvc-cluster.xml

```diff
@@ -51,7 +51,7 @@ limitations under the License.
     <attribute side="server" code="0x0001" define="CURRENT_PHASE"           type="INT8U"                                    min="0x00" max="0x1F"    writable="false" isNullable="true"  optional="false">CurrentPhase</attribute>
     <attribute side="server" code="0x0002" define="COUNTDOWN_TIME"          type="elapsed_s"                                min="0"    max="259200"  writable="false" isNullable="true"  optional="true">CountdownTime</attribute>
     <attribute side="server" code="0x0003" define="OPERATIONAL_STATE_LIST"  type="ARRAY" entryType="OperationalStateStruct"                          writable="false"                    optional="false">OperationalStateList</attribute>
-    <attribute side="server" code="0x0004" define="OPERATIONAL_STATE"       type="OperationalStateStruct"                                            writable="false"                    optional="false">OperationalState</attribute>
+    <attribute side="server" code="0x0004" define="OPERATIONAL_STATE"       type="ENUM8"                                                             writable="false"                    optional="false">OperationalState</attribute>
```

Comment:
Here I guess ENUM8 makes sense since it might be the OperationalStateEnum from this cluster of from OperationalState.  But please add a comment in the XML explaining that?  Because once we start generating the XML from the spec (or vice versa), this bit is going to be a problem....

Reply:
Sure, add a comment.